Libraries/Share/Share.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) 2016-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule Share
+ * @flow
+ */
+'use strict';
+
+const Platform = require('Platform');
+const {
+  ActionSheetManager,
+  ShareModule
+} = require('NativeModules');
+const invariant = require('fbjs/lib/invariant');
+const processColor = require('processColor');
+
+class Share {
+  
+  /**
+   * Open a dialog to share text content.
+   * 
+   * ### Content
+   * 
+   *  - `message` - a message to share
+   *  - `url` - an URL to share. In Android, this will overwrite message
+   *  - `subject` - subject of the message
+   * 
+   * At least one of URL and message is required.
+   * 
+   * ### Options
+   *
+   * #### iOS
+   *
+   * - `excludedActivityTypes`
+   * - `tintColor`
+   *
+   * #### Android
+   * 
+   * - `dialogTitle`
+   */
+  static shareText(content: Object, options?: Object): Promise<boolean> {
```

Comment:
Can you change `subject` to `title` instead?

Also it'll be great to have Flow Types for content and options

```js
type Content = { title: string, message: string } | { title: string, url: string };
type Options = { dialogTitle?: string, excludeActivityTypes?: boolean, tintColor?: string };
```

BTW what does `excludeActivityTypes` do?